packages/forms/src/Components/DateTimePicker.php

```diff
@@ -38,6 +38,8 @@ class DateTimePicker extends Field
 
     protected string | Closure | null $timezone = null;
 
+    protected string | Closure | null $icon = 'default';
```

Comment:
`null` should be default.
Disabling should be done by accepting a bool `false`.